src/docs/product/releases/index.mdx

```diff
@@ -54,6 +54,23 @@ _Release health_ provides insight into the impact of crashes and bugs as it rela
 
 You can view release health data either from the <SandboxLink scenario="oneIssue" projectSlug="react">Issue Details</SandboxLink> page by selecting the commit ID listed under Last Seen", or from the **Releases** page.
 
+## Resolve in release
+
+![View of the resolve issue dropdown options on issue details.](resolved_in_options.png)
+
+Marking an issue as [resolved](/product/issues/states-triage/#resolve) in a release or in the next release allows you to mark an issue as resolved until the issue is seen again in a future release.
+
+If you'd like the issue to be marked as unresolved the next time an event is seen in any release, use the main "Resolve" button.
+If you'd like to ignore events until an event is seen in a future release, you can mark it resolved in the current release or the next release.
+
+**Resolved in the current release** records the current release, displayed in the UI and new events for this issue are compared with the recorded release version to determine if the issue should unresolve. If the release version is using semver, the issue will be marked as unresolved if the release is from a semver version that is greater than the resolved version. If the release is not using semver, the release creation dates are compared.
+
+**Resolved in the next release** works similar to resolved in current release except that we can't record the resolution version since it hasn't been created yet. Instead, we record the current release version and update it when a newer release is created.
+
```

Comment:
```suggestion
### Resolved in the Next Release 

The "Resolved in the next release" option works in a similar way to resolved in the current release. But because we can't record the resolution version since it hasn't been created yet, we record the current release version and update it when a newer release is created.

```